### PR TITLE
feat(html/parser): `raw` for text

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Support `raw` for future linting and allow to get original written value in codegen.

**BREAKING CHANGE:**

Yes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_html_parser/tests/recovery/element/a/input.html [hunk 1]
```diff
@@ -1 +1 @@
-<a><p></a></p>
\ No newline at end of file
+<
```
Comment (kdy1) [inline crates/swc_html_parser/tests/recovery/element/a/input.html:1]: Is this change intentional?

Comment (alexander-akait) [reply crates/swc_html_parser/tests/recovery/element/a/input.html:1]: Good catch, thank you

**Comment by alexander-akait** [comment]
@kdy1 Fixed

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_html_ast --breaking
